### PR TITLE
Fixed requested changes for the bachelorproject's team page

### PR DESCRIPTION
**Pull request by odanivan**
- Ivan Ilic's image is correct circle
- Johannes Unruh has image placeholder
- Marco Schaarschmidt's name is displayed with less letter-spacing to fit inside box without line break
:)

### CODE REVIEW

**Comment by schurigh** [comment]
😃👍

File: views/team/team.hbs [hunk 1]
```diff
@@ -26,7 +26,7 @@
                                             </div>
                                         </div>
                                         <div class="col-md-7">
-                                            <h2 class="person-title" style="text-align: left">{{title}} {{firstName}} {{lastName}}</h2>
+                                            <h2 class="person-title" descr="{{lastName}}" style="text-align: left">{{title}} {{firstName}} {{lastName}}</h2>
```
Comment (adrianjost) [inline views/team/team.hbs]:
```suggestion
                                            <h2 class="person-title" data-lastname="{{lastName}}" style="text-align: left">{{title}} {{firstName}} {{lastName}}</h2>
```
`descr` isn't a valid html attribute. Please use `data-lastname` or something similar.

File: views/team/team.hbs [hunk 2]
```diff
@@ -48,7 +48,7 @@
                             <div class="card h-100 person">
                                 <img class="card-img-top circular" src="{{pictureUrl}}">
                                 <div class="card-block">
-                                    <h4 class="person-title card-title">{{firstName}} {{lastName}}</h4>
+                                    <h4 class="person-title card-title" descr="{{lastName}}">{{firstName}} {{lastName}}</h4>
```
Comment (adrianjost) [inline views/team/team.hbs]:
```suggestion
                                    <h4 class="person-title card-title" data-lastname="{{lastName}}">{{firstName}} {{lastName}}</h4>
```

File: static/styles/team/team.scss [hunk 3]
```diff
@@ -159,6 +159,9 @@
     color: #000;
     font-size: 2em;
   }
+  .person-title[descr="Schaarschmidt"] {
```
Comment (adrianjost) [inline static/styles/team/team.scss]:
```suggestion
  .person-title[data-lastname="Schaarschmidt"] {
```

**Pull request merged**